README.md

```diff
@@ -64,6 +64,10 @@ If you run your SIMD accelerated code on an unsupported platform, WebAssembly wi
 
 On other platforms than WASM, this feature does nothing and RustFFT will behave like it is not set.
 
+### std
+
+The scalar implementation (no avx, sse, neon or wasm SIMD) is available for no-std targets. To build the crate as no-std simply disable default fetures.
+
```

Comment:
Currently you also need to enable `libm` feature. Would be cool if `default-features = false` implies `libm` or something. Same with `primal-check` crate. I'm not really sure what  the generally acceptable thing to do is in Rust but if it's not possible to express "either `std` or `libm`" needs to be enabled, then you should add documentation on enabling `libm` here.